src/media.cpp

```diff
@@ -290,6 +292,7 @@ FeVideoImp::FeVideoImp( FeMedia *p )
 		m_video_thread( &FeVideoImp::video_thread, this ),
 		m_parent( p ),
 		run_video_thread( false ),
+		sws_flags( SWS_FAST_BILINEAR ),
```

Comment:
Hi h0tw1r3, thanks for this!

I think you need to initialize sws_ctx to NULL in the FeVideoImp constructor, in order for the check in the destructor to work.  Could you add that and then I can merge your changes? Thanks!!